media/server/gstplayer/include/tasks/generic/AttachSamples.h

```diff
@@ -27,7 +27,7 @@
 #include <gst/gst.h>
 #include <vector>
 
-namespace firebolt::rialto::server
+namespace firebolt::rialto::server::generic
```

Comment:
This namespace is a little bit confusing for me. generic here is only for tasks, not whole rialto server. If you want to introduce a new namespace here, I would use rialto::server::tasks::generic/webaudio

Reply:
Yeah that looks clearer, I can make that change.